rocketmq-replicator/pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <groupId>org.apache.rocketmq</groupId>
     <artifactId>rocketmq-connector</artifactId>
```

Comment:
Could we change the artifactId to  "rocketmq-replicator" ?

Reply:
Sure，no problem

Reply:
> Could we change the artifactId to "rocketmq-replicator" ?

sure, no problem